src/Core/src/Platform/Android/ContentViewGroup.cs

```diff
@@ -109,7 +109,7 @@ void ClipChild(Canvas? canvas)
 
 			float density = Context.GetDisplayDensity();
 
-			float strokeThickness = (float)Clip.StrokeThickness;
+			float strokeThickness = (float)(Clip.StrokeThickness * density);
 			float offset = strokeThickness / 2;
 			float w = (canvas.Width / density) - strokeThickness;
 			float h = (canvas.Height / density) - strokeThickness;
```

Comment:
Is this correct? `strokeThickness` is now multiplied by `density` but `canvas.Width` / `canvas.Height` is divided by `density`.

Reply:
We apply a Scale by density after this.